edx/edx-ora2
# Increase character limit for Instructor feedback on ORAs

**TL;DR -** Increase character limit for Instructor feedback on ORAs.

JIRA: [EDUCATOR-5628](https://openedx.atlassian.net/browse/EDUCATOR-5628)

**What changed?**

- Increase the frontend character limit in feedback templates from 1k to 10k.

**Note:** the UI looks kind of gross and limits the horizontal space this feedback can take up. I wrote up a separate ticket ([EDUCATOR-5805](https://openedx.atlassian.net/browse/EDUCATOR-5805)) to change it.

**Developer Checklist**


**Testing Instructions**

1. Install and run migrations
2. In both a staff-graded ORA and peer-graded ORA, submit an ORA response as a student.
3. Provide assessments (as staff or peer) with between 1k and 10k characters to complete the ORA grading workflow.
4. View the "My Grade" section as a student, verify that the feedback sections are not clipped.
5. Attempt to provide an assessment with > 10k characters. Frontend should limit input to 10k characters (browser dependent). I've attached [a text file with just over 10k characters](https://github.com/edx/edx-ora2/files/6517105/large-feedback.txt) for easy testing. The trailing `THIS_SHOULD_BE_TRIMMED` should not appear.

Note: it is possible to bypass this on the frontend but the database can support much longer inputs.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
